qiskit/transpiler/passes/scheduling/scheduling/alap.py

```diff
@@ -20,8 +20,8 @@
 class ALAPScheduleAnalysis(BaseScheduler):
     """ALAP Scheduling pass, which schedules the **stop** time of instructions as late as possible.
 
-    See :class:`~qiskit.transpiler.passes.scheduling.base_scheduler.BaseScheduler` for the
-    detailed behavior of the control flow operation, i.e. ``c_if``.
+    See :class:`~qiskit.transpiler.passes.scheduling.scheduling.base_scheduler.BaseScheduler` for
+    the detailed behavior of the control flow operation, i.e. ``c_if``.
```

Comment:
I don't think this (or any similar) cross-ref will resolve; `BaseScheduler` isn't included in any autosummary directive.

Reply:
It's not, I noticed that too after I pushed it. This was existing before I just mechanically updated the paths for things. I'm not sure how we want to handle this, since to include it in the doc tree we'd probably need to re-export the class to `qiskit/transpiler/passes/__init__.py` but I assume @nkanazawa1989 didn't do that intentionally.

@nkanazawa1989 do you have any preference on how you want to include the `BaseScheduler` class in the docs?

Reply:
Umm... I wrote detailed document about the policy of classical IO handling there, and I don't want to copy the document for every child scheduler pass. Because `BaseScheduler` is an abstract class, I think we don't need to expose this to module level import, but I want to provide this docs with users. Perhaps we can move document to https://qiskit.org/documentation/apidoc/transpiler.html and replace the link?

Reply:
I added a scheduling section to the top level transpiler docs and included these details there in: https://github.com/Qiskit/qiskit-terra/pull/7884/commits/3ae60d20bec3f5e41156468248dc4a6068f525b5